src/documents/InclusiveMakingAndHacking.html.md.handlebars

```diff
@@ -0,0 +1,205 @@
+
+## Inclusively Designing Hackathons and Hacking
+
+It's been said about hackathons that the connections and ideas that get made between people attending are more important than any products that get produced. If hackathons are about building community, issues of inclusion, exclusion and audience diversity are paramount to their design.
+
+In thinking about inclusive design for hackathons, we must ask ourselves some of the same questions we ask in other contexts:
+
+### Asking Who Is Excluded
+
+We must consider <a href="WhatIsTheApproach.md#whoIsExcluded">who might be excluded?</a>, and how to redesign our approach to running, promoting and recruiting for our events.
+
+This is especially critical for hackathons trying to tackle social challenges faced by identifiable groups such as people with disabilities, under the principle of <a href="https://en.wikipedia.org/wiki/Nothing_About_Us_Without_Us" target="_blank" class="link-external">*Nothing About Us Without Us*</a>. Any extra work involved in broadening the reach of the event beyond the "natural" demographic will be balanced out by better outcomes and learning.
+
+### Expanding Collaboration
+
+Physical colocation is common with hackathons, but a requirement to be physically present to participate can exclude people based on ability to travel to the location or by disability.
+
+When considering the design of a hackathon-style event, consider:
+
+* Can you support remote collaboration using audio and video conferencing tools, shared document workspaces, and/or collaborative coding environments?
+* Are there alternate styles of event that would also be appropriate, such as the distributed approach of the <a href="http://globalgamejam.org/" target="_blank" class="link-external">Global Game Jam</a>?
+
+### The Curb Cut Effect and Innovation from the Margins
+
+There are a number of practical reasons to engage with individuals with "extreme" needs that falls outside the "average", along with the moral imperatives of inclusion.
+
+First, there are no "average" users in inclusive design's philosophy. Per the first dimension of inclusive design, <a href="http://idrc.ocadu.ca/index.php/resources/idrc-online/library-of-papers/443-whatisinclusivedesign" target="_blank" class="link-external">"As individuals spread out from the hypothetical average, the needs of individuals that are outliers, or at the margins, become ever more diverse. Most individuals stray from the average in some facet of their needs or goals."</a>
+
+Secondly, designs intended to serve the needs or goals of individuals on the margin can turn out to be of much broader use. This is sometimes referred to as the *curb-cut effect* after the sidewalk ramps that were originally intended for wheelchair users, but have ended up being of significant benefit to many other users of public spaces.
+
+### Considering the Full Range of Human Diversity
```

Comment:
I think it might be helpful to move this section further up, since it gets at some of the questions I raised in the previous section.